drivers/dma/Kconfig.mcux

```diff
@@ -0,0 +1,27 @@
+# DMA configuration options
+
+# Copyright (c) 2020, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config DMA_MCUX
+	bool "Enable MCUX DMA driver"
+	depends on SOC_SERIES_KINETIS_K6X
```

Comment:
Change this to `depends on HAS_MCUX_EDMA` and add an EDMA entry to `modules/Kconfig.mcux`. You'll then need to add `select HAS_MCUX_EDMA` to `soc/arm/nxp_imx/rt/Kconfig.soc` and `soc/arm/nxp_kinetis/k6x/Kconfig.soc`

Reply:
ok

Reply:
done